NeuroID/NeuroIDClass/NeuroID.swift

```diff
@@ -72,6 +72,9 @@ public enum NeuroID {
 
     internal static var CURRENT_ORIGIN: String?
     internal static var CURRENT_ORIGIN_CODE: String?
+    
+    private static var lowMemory: Bool = false
+
```

Comment:
If we are just using this internally, why don't we make this internal and then use it directly instead of creating it private and then creating getters/setters that have an internal scope?

Reply:
I wanted to have getters in setters so that it gives us a bit more flexibility in case we want to do other operations later. 

Yea, should def be internal

Reply:
You mean other operations on the setting of low memory? If that is the case why don't we make a private setter here with an internal get and then we can use the setLowMemory internal function you have? We still reduce the extra functions by 1